### DISCUSSION TITLE
Disable shortcut on boot MIUI

### DISCUSSION THREAD

**Discussion by xDoge26** [LSPosed/LSPosed#1747] [Q&A] [answered]
How do i disable create shortcut on boot ?

**Comment by yujincheng08** [comment, marked as answer]
In setting.

**Reply by xDoge26** [reply]
i toggle off force app show icon but it still show

**Reply by yujincheng08** [reply]
not that one

**Reply by xDoge26** [reply]
send me a screenshot please , i cant find it

**Reply by davdklkn** [reply]
You have to download the LSPosed app from Google Play. I found the setting there. It can be uninstalled after.

**Reply by xDoge26** [reply]
> You have to download the LSPosed app from Google Play. I found the setting there. It can be uninstalled after.

thanks

**Reply by ghost** [reply]
> You have to download the LSPosed app from Google Play. I found the setting there. It can be uninstalled after.

in google play there is an old version.
You must install manager.apk from command line
open a shell as root and write this command

`pm install /data/adb/lspd/manager.apk`

in setting you see this

![IMG_20220317_170231](https://user-images.githubusercontent.com/3065939/158843086-c0337f73-c652-4b2a-9a63-4489cedf5fd2.jpg)

is available only when the app is not parasitic
